### PR TITLE
Update notebook to properly run

### PR DESCRIPTION
**Pull request by georgf**
Update notebook to:
- not error out on current EMR git state
- just remove existing code, cache & output dirs to avoid stale states

### CODE REVIEW

File: notebooks/load_and_run.ipynb [hunk 1]
```diff
@@ -47,6 +47,51 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "!git config --global user.email \"gfritzsche@mozilla.com\" && \\\n",
```
Comment (Dexterp37) [inline notebooks/load_and_run.ipynb]: Is this command intentional or a leftover? If the former, why is this needed?

Comment (georgf) [inline notebooks/load_and_run.ipynb]: This is intentional.
git was erroring out on our current machines without it.
